exp-player/addon/components/exp-lookit-geometry-alternation/component.js

```diff
@@ -550,7 +550,7 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
          *
          * @event startIntro
          */
-        _this.sendTimeEvent('startIntro');
+        _this.send('setTimeEvent', 'exp-alternation:startIntro');
```

Comment:
@kimberscott : With the new `makeTimeEvent`, this means that the event name will be:

`exp-alternation:exp-alternation:startIntro`

Is that intentional? I left some usages in old frames like this, but refactoring hinges on whether there is already live data  / an analysis pipeline for studies using this frame. New frames may give us more leeway.

Willing to make the change, but I run any changes to live experiments by you first.